### PR TITLE
Bump to lts-9.11 with backwards compatibility

### PR DESCRIPTION
**Pull request by stites**
note that I can't actually build with GHC-8.0.1 (I get a weird error I don't have time to debug) so https://github.com/NorfairKing/super-user-spark/pull/22/files#diff-94b985dee807af6c223c09f638bb611eR53 should be verified on a lts-7.11 build.

### CODE REVIEW

**Comment by stites** [comment]
I allowed failure on nightly since there isn't a stable lts for GHC-8.2.

File: src/Arguments.hs [hunk 1]
```diff
@@ -50,6 +50,9 @@ runOptionsParser strs = execParserPure prefs optionsParser strs
       { prefMultiSuffix = "SPARK"  -- metavar suffix for multiple options
       , prefDisambiguate = True    -- automatically disambiguate abbreviations (default: False)
       , prefShowHelpOnError = True -- always show help text on parse errors (default: False)
+#if MIN_VERSION_base(4,9,1)
```
Comment (NorfairKing) [inline src/Arguments.hs]: I think you want a `MIN_VERSION` of `optparse-applicative` instead.

Comment (stites) [reply src/Arguments.hs]: oh! I didn't know you could do that. I'll fix this in a bit.

**Comment by stites** [comment]
bump! now using the optparse_applicative pragma

**Comment by NorfairKing** [comment]
Thanks!!

**Pull request merged**